apps/spruce/src/components/404/NotFoundSvg.tsx

```diff
@@ -1,13 +1,1269 @@
-import notFound from "./notFound.svg";
-
-// It's not possible to lazy load an SVG, so we wrap the SVG in a React component.
-const NotFoundSvg: React.FC = () => (
-  <img
-    alt="Page not found"
-    data-cy="404"
-    src={notFound}
-    style={{ height: "inherit", width: "100%", objectFit: "cover" }}
-  />
-);
+import { useEffect, useRef } from "react";
+
+interface NotFoundSvgProps {
+  disableAnimations?: boolean;
+}
+
+const NotFoundSvg: React.FC<NotFoundSvgProps> = ({
+  disableAnimations = false,
+}) => {
+  const svgRef = useRef<SVGSVGElement>(null);
+
+  useEffect(() => {
```

Comment:
Apparently Chromatic is supposed to [automatically disable](https://www.chromatic.com/docs/animations/) animations; is the useEffect still necessary with the updated SVG element?